ref-exchange/src/account_deposit.rs

```diff
@@ -26,8 +26,8 @@ pub struct AccountDeposit {
 
 impl AccountDeposit {
     /// Adds amount to the balance of given token while checking that storage is covered.
-    pub fn add(&mut self, token: AccountId, amount: Balance) {
-        if let Some(x) = self.tokens.get_mut(&token) {
+    pub(crate) fn add(&mut self, token: &AccountId, amount: Balance) {
```

Comment:
this functions doesn't need to be exposed.
Also, by using references we avoid copies.